### PR TITLE
Wiki reorg: Wallets

### PR DESCRIPTION
**Pull request by filippoweb3**
See [this issue](https://github.com/w3f/polkadot-wiki/issues/4376) and [this google doc](https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit) for more information about the Wiki reorg.

This PR also focuses on reorganizing the sidebar (Section "General").

### CODE REVIEW

File: docs/general/ledger.md [hunk 1]
```diff
@@ -1,53 +1,61 @@
...
-sidebar_label: Ledger Application
+sidebar_label: Ledger
 description: How to use the Polkadot application on Ledger.
 keywords: [ledger, staking, polkadot app]
-slug: ../learn-ledger
+slug: ../ledger
 ---
 
-:::info
+:::info Supported Wallet
```
Comment (profradha) [inline docs/general/ledger.md]:
```suggestion
:::info Need support with your Ledger Wallet?
```

File: docs/general/ledger.md [hunk 2]
```diff
@@ -1,53 +1,61 @@
...
+:::warning Please Note
+
+- Because of required WebUSB support, Ledger wallets currently only work on Chromium-based browsers
+  like Google Chrome.
 
-Ledger does not currently support batch calls; As a consequence, if you stake using the
-[staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
-Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that allows
-for single calls (i.e., bonding and nominating).
+- Ledger does not currently support batch calls. As a consequence, if you nominate using the
```
Comment (profradha) [inline docs/general/ledger.md]: Is this still true? I see it as a supported feature on Zondax Polkadot app repo https://github.com/Zondax/ledger-polkadot#utility

File: docs/general/ledger.md [hunk 3]
```diff
@@ -1,53 +1,61 @@
...
+  [staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
+  Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that
+  allows for single calls (i.e., bonding and nominating). **Pool members are not affect by this
+  issue.**
```
Comment (profradha) [inline docs/general/ledger.md]:
```suggestion
  allows for single calls (i.e., bonding and nominating). **Pool members are not affected by this
  issue.**
```

File: docs/general/ledger.md [hunk 4]
```diff
@@ -1,53 +1,61 @@
...
-usually developed by third parties, and they enable the user to transact securely on the blockchain
-network. The Polkadot Ledger application allows you to manage Polkadot's native token, DOT. It
-supports most of the transaction types of the network, including batch transactions from the Utility
-pallet.
+Ledger devices can be equipped with applications that are blockchain-specific. Third parties usually
+develop such applications, enabling the user to transact securely on the blockchain network. The
+Polkadot Ledger application allows you to manage
+{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} native token,
+{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}. It supports most of the transaction types of
+the network, including batch transactions from the Utility pallet.
```
Comment (profradha) [inline docs/general/ledger.md:41]: Here we are confirming that batch transactions from Utility pallet are supported.

File: docs/general/ledger.md [hunk 5]
```diff
@@ -1,53 +1,61 @@
...
+- Because of required WebUSB support, Ledger wallets currently only work on Chromium-based browsers
+  like Google Chrome.
 
-Ledger does not currently support batch calls; As a consequence, if you stake using the
-[staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
-Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that allows
-for single calls (i.e., bonding and nominating).
+- Ledger does not currently support batch calls. As a consequence, if you nominate using the
+  [staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
+  Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that
```
Comment (profradha) [inline docs/general/ledger.md]: If batch calls are supported, this content needs modification

File: docs/general/ledger.md [hunk 6]
```diff
@@ -1,53 +1,61 @@
...
-others are not supported at all by any version. For instance, the staking `rebag` extrinsic is
-supported on the XL version but not on the light version. Support for the XL version is limited to
-single calls (not batch calls), meaning you can rebag one account at a time. Also, joining a
-[nomination pool](../learn/learn-nomination-pools.md) is only possible with the XL version.
+Check the
+{{ polkadot: [Ledger Polkadot App](https://github.com/Zondax/ledger-polkadot) :polkadot }}{{ kusama: [Ledger Kusama App](https://github.com/Zondax/ledger-kusama) :kusama }}
+specification for the list of transactions supported. Some transactions are supported only on a
+specific app version, and others are not supported by any version. For instance, the staking `rebag`
+extrinsic is supported on the XL version but not on the light version. Support for the XL version is
+limited to single calls (not batch calls), meaning you can rebag one account at a time. Also,
```
Comment (profradha) [inline docs/general/ledger.md]: Batch calls are supported by all versions of the apps https://github.com/Zondax/ledger-polkadot#utility

File: docs/general/ledger.md [hunk 7]
```diff
@@ -194,23 +232,24 @@ Democracy extrinsics on the full ledger.
...
+Ledger apps for the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} ecosystem are
+developed by [Zondax](https://zondax.ch/). When new functionalities are added to the Ledger apps,
+they are made available on a developer release for testing purposes. After a successful audit and
+review, the apps would be available for download and installation using
+[Ledger Live](https://www.ledger.com/ledger-live). As it takes some time for Ledger to audit and
+review the release, the app upgrade option may not be available on Ledger Live when the new runtime
+is deployed on the network. If this happens, users cannot use Ledger devices with the Polkadot-JS
+UI, and while signing for a transaction, will most likely incur the error message "txn version not
+supported". Please do not panic if this happens, as there are solutions to this problem. Suppose you
+cannot wait a few days until the app passes the Ledger audit. In that case, you can install the
```
Comment (profradha) [inline docs/general/ledger.md]:
```suggestion
cannot wait a few days until the app passes the Ledger audit, you can install the
```

File: docs/general/polkadotjs-ui.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
+---
+id: polkadotjs-ui
+title: Polkadot-JS UI (Wallet)
+sidebar_label: Polkadot-JS UI
+description: Learn about PolkadotJS
+keywords: [polkadotjs, polkadotjs apps, apps UI, extension]
+slug: ../polkadotjs-ui
+---
+
+:::info Supported Wallet
```
Comment (profradha) [inline docs/general/polkadotjs-ui.md]:
```suggestion
:::info A Wallet for Developers and Power Users
```

**Review by DrW3RK — COMMENTED** [review]
I will review this is phases. Here is the feedback on the Ledger section of the PR

File: docs/general/ledger.md (hunk 2 above)
Comment (filippoweb3) [reply docs/general/ledger.md]: removed

File: docs/general/ledger.md (hunk 5 above)
Comment (filippoweb3) [reply docs/general/ledger.md]: removed

File: docs/general/ledger.md (hunk 6 above)
Comment (filippoweb3) [reply docs/general/ledger.md]: yep, content removed

**Comment by filippoweb3** [comment]
@DrW3RK thanks for the review on the Ledger page, I addressed it and you can proceed to the other pages ;)

File: docs/general/polkadotjs-ui.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+### Governance
+
+In this section of the UI, you can participate in [**governance**](../learn/learn-opengov.md).
+
+### Developer
+
+In this section of the UI, you can access advanced features such as:
+
+- [Query chain metadata](https://polkadot.js.org/apps/#/chainstate)
```
Comment (profradha) [inline docs/general/polkadotjs-ui.md]: On-chain data is more intuitive 

```suggestion
- [Query on-chain data](https://polkadot.js.org/apps/#/chainstate)
```

File: docs/general/polkadotjs-ui.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+- [Query on-chain data using RPC calls](https://polkadot.js.org/apps/#/rpc)
```
Comment (profradha) [inline docs/general/polkadotjs-ui.md]:
```suggestion
- [Query the node using RPC calls](https://polkadot.js.org/apps/#/rpc)
```

File: docs/general/polkadotjs.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+The extension is a robust key-storage tool, i.e. if you clear the cache of your browser your
+accounts will be kept. This does not happen if you added an account directly to the Polkadot-JS UI.
+Also, the extension recognizes websites that have been flagged for malicious activity. Another
+security note, the extension will always ask you if you want a specific website to access the
+account information in it.
+
+:::info Metadata Updates
+
+The browser extension is a tool that interacts with the Polkadot network, but it disconnected from
+it when you is not in use. Is is important you
```
Comment (profradha) [inline docs/general/polkadotjs.md]:
```suggestion
The browser extension is a tool that interacts with the Polkadot network, but it is disconnected from
it when it is not in use. It is important you
```

File: docs/general/polkadotjs.md [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+[**always check for metadata updates**](../learn/learn-extrinsics.md#metadata-updates) before using
+the extension other account management tools such as Parity Signer (aka Polkadot Vault).
```
Comment (profradha) [inline docs/general/polkadotjs.md]:
```suggestion
the extension or other account management tools such as Parity Signer (aka Polkadot Vault).
```

File: docs/general/polkadotjs.md [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+
+:::
+
+## Polkadot-JS Phishing List
+
+The [**Polkadot-JS phishing list**](https://polkadot.js.org/phishing/) website is a community-driven
+curation of less-than-honest operators. The Polkadot-JS extension uses this list to warn a user
```
Comment (profradha) [inline docs/general/polkadotjs.md]:
```suggestion
curation of malicious actors and operators. The Polkadot-JS extension uses this list to warn a user
```

File: docs/general/polkadotjs.md [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+about suspicious URLs and addresses that are part of the list, and automatically blocks the account
+address.
+
+## Polkadot-JS API
+
+The [**Polkadot-JS API**](https://github.com/polkadot-js/api) is a JavaScript API allowing for
+programs to interface with the functionality of
```
Comment (profradha) [inline docs/general/polkadotjs.md]:
```suggestion
programs to interface with the functionalities of
```

Comment (profradha) [inline docs/general/polkadotjs.md]:
```suggestion
programs to interface with the functionalities of
```

File: docs/general/scams.md [hunk 15]
```diff
@@ -1,7 +1,7 @@
 ---
 id: scams
 title: How to Protect Yourself from Scams
-sidebar_label: How to Protect Yourself from Scams
+sidebar_label: Scam Prevention
```
Comment (profradha) [inline docs/general/scams.md]:
```suggestion
sidebar_label: Scam Protection
```

File: docs/general/wallets.md [hunk 16]
```diff
@@ -7,40 +7,43 @@ keywords: [wallets, account, signer, supported wallets, balance, token]
...
+public-private key pair. Access to a private key gives full access to all the allowed transactions
+on that blockchain account. It is essential to keep the private key secure.
 
-## Non-Custodial Wallets
+Typically, the account keys are either stored and accessed through a browser extension or a
+smartphone app (which are considered as a hot wallets as they are online), or an air-gapped device
+or a hardware wallet (which are considered as cold wallets as they are offline).
 
-:::caution
+:::tip Not your keys, not your coins!
```
Comment (profradha) [inline docs/general/wallets.md]: Still prefer tokens instead of coins here. None of the tokens are referred to as coins in Polkadot ecosystem.

**Review by DrW3RK — APPROVED** [review]
Minor fixes suggested.

**Pull request merged**